### ISSUE TITLE
Timezone

### ISSUE THREAD

**Issue by Sorunome** [toxygen-project/toxygen#36]
The time is off by one hour, the OS displays me 15:06 and toxygen 16:06. I'm in the timezone GMT+1. Perhaps toxygen isn't aware of summer/winter time, as I don't recall it being off during summer?

**Comment by ingvar1995** [comment]
>as I don't recall it being off during summer?

**Comment by Sorunome** [comment]
In the sumer the time offset was correctly as i'm in GMT+2 then, but now, during winter, the timezone is GMT+1 but it still displays as if it was GMT+2

**Comment by ingvar1995** [comment]
Can you please provide output for:
```
python3
import time
time.localtime().tm_isdst
```

**Comment by Sorunome** [comment]
```
sorunome@sorunome-desktop ~ $ python3                                                                                                                                                                                                     130
Python 3.6.0 (default, Dec 24 2016, 08:03:08) 
[GCC 6.2.1 20160830] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import time
>>> time.localtime().tm_isdst
0
```

**Comment by ingvar1995** [comment]
Should be fixed in 9b4965d
